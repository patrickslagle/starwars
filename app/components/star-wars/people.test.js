import React from 'react'
import App from './app.jsx'
// import renderer from 'react-test-renderer'
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';

configure({ adapter: new Adapter() });

test('App component renders properly', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});