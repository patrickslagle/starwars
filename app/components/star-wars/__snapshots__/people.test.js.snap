// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders properly 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <h1>
    Star Wars People
  </h1>
  <SearchPeople
    fetchPeople={[Function]}
  />
  <div
    id="people-container"
  />
  <FooterNav
    fetchPeople={[Function]}
    loadingPeople={true}
    next={null}
    prev={null}
  />
</div>
`;
